VMT/VMT.md

```diff
@@ -0,0 +1,225 @@
+> be involved in.
+
+The description is validated by the reporter and the [Architecture Committee](https://github.com/kata-containers/community#architecture-committee).
+
+### Send CVE request
+
+To ensure full traceability, we attempt to obtain a CVE assignment before the issue is communicated to a larger public.
+This is generally done as the patch gets nearer to final approval.
+
+> **FIXME** I'm presuming we will obtain CVEs for our vulnerabilities - are we all happy using this service?
+
+The KCSA Issue status is set to *In progress* and the approved impact description is submitted through [MITRE's CVE Request form](https://cveform.mitre.org/).
+The *request type* is `Request a CVE ID`, the *e-mail address* should be that of the requester (generally the assigned VMT coordinator in the case of reports officially managed by the VMT), and for embargoed reports the coordinator's OpenPGP key should be pasted into the field provided.
+
+In the *required* section set the checkboxes indicating the product is not [CVE Numbering Authority](https://cve.mitre.org/cve/cna.html) (CNA) covered and that no prior CVE ID has been assigned, select an appropriate *vulnerability type* (using `Other or Unknown` to enter a freeform type if there is nothing relevant on the drop-down), set the *vendor* to `Kata Containers`, and the *product* and *version* fields to match the `$PROJECTS` and `$AFFECTED_VERSIONS` from the impact description.
+In the *optional* section set the radio button for *confirmed/acknowledged* to `Yes`, choose an appropriate *attack type* in the drop-down (often this is `Context-dependent` for our cases), check the relevant *impact* checkboxes, attempt to fill in the *affected components* and *attack vector* fields if possible, paste in the *suggested description* from the prose of the impact description (usually omitting the first sentence as it's redundant with other fields), put the `$CREDIT` details in the *discoverer/credits* field, and the Issue URL (along with the public Issue and PR URLs for patches if already public) in the *references* field.
+If the report is still private, note that in the *additional information* field like `This report is currently under embargo and no disclosure date has been scheduled at this time.`
+
+At the bottom of the page, fill in the *security code* and click the *submit request* button.
+If some fields contain invalid data they will be highlighted red; correct these, update the *security code* and *submit request* again until you get a confirmation page.
+
+## Get assigned CVE
+
+MITRE returns the assigned CVE.
+It is added to the GitHub Issue in the comment field, and the Issue is retitled to "$TITLE ($CVE)".
+
+## Embargoed disclosure
+
+Once the patches are approved and the CVE is assigned, a signed email with the vulnerability description is sent to the downstream stakeholders.
+The disclosure date is set to 3-5 business days, excluding Monday/Friday and holiday periods, at 1500 UTC.
```

Comment:
- Sorry to be pedantic, but holiday periods for which country? I can only imagine "all" but how to know if a particular day is a holiday period somewhere on Earth... ;)
- Why 3pm?

Reply:
Looks like this was copied from OpenStack's VMT process. We chose 1500 UTC years ago mainly because it was a convenient time for the particular people on the VMT who were coordinating advisory publication. It's a loose guideline so that any downstream stakeholders who were notified under embargo can know roughly when to expect they'll be able to publish their fixed packages, product-specific ports of patches, or update behavior of their public-facing service endpoints without breaking embargo.

What's important is that you pick a date and time, and the more you standardize about that choice the less is left up to chance later reducing the opportunity for uncoordinated/premature disclosure.

Reply:
Pendantic? ;-)
I think the implication is 'flexibility where needed on availability of the VMT members and the downstream stakeholders'. We probably don't want to make that 100% formal.
As @fungi says, I think it is important we pick a date and time, and if we can try to unify what that date and time will be so folks are used to the process and know what to expect, that is better.
Presently this project is mostly split across the West Coast U.S.A and Asia I would say? In which case, I think the best time would be late PST time/early CST I think? How about 0000 UTC ?